webextensions/api/scripting.json

```diff
@@ -0,0 +1,393 @@
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "safari_ios": {
+                "version_added": "15.4",
+                "notes": "Available for use in Manifest V3 only."
+              }
+            }
+          }
+        },
+        "InjectionTarget": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/scripting/InjectionTarget",
+            "support": {
+              "chrome": {
+                "version_added": "88",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "edge": {
+                "version_added": "88",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "firefox": {
+                "version_added": "101",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "firefox_android": {
+                "version_added": "101",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "opera": {
+                "version_added": "74",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "safari": {
+                "version_added": "15.4",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "safari_ios": {
+                "version_added": "15.4",
+                "notes": "Available for use in Manifest V3 only."
+              }
+            }
+          }
+        },
+        "RegisteredContentScript": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/scripting/RegisteredContentScript",
+            "support": {
+              "chrome": {
+                "version_added": "88",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "edge": {
+                "version_added": "88",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "firefox": {
+                "version_added": "101",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "firefox_android": {
+                "version_added": "101",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "opera": {
+                "version_added": "74",
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "safari": {
+                "version_added": null,
+                "notes": "Available for use in Manifest V3 only."
+              },
+              "safari_ios": {
+                "version_added": null,
+                "notes": "Available for use in Manifest V3 only."
+              }
+            }
+          }
+        },
+        "executeScript": {
```

Comment:
`executeScript` needs extra sections on specific properties:

- `injectImmediately`
   - not supported by Safari
- `world`
   - not supported by Safari
   - Firefox only supports ISOLATED, not MAIN.

Reply:
Updated